models/Recipe.model.js

```diff
@@ -2,7 +2,15 @@ const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
 
 const recipeSchema = new Schema({
-  // TODO: write the schema
+  title:{type:String,require: true, unique:true},
+  level:{type:String,enum:['Easy Peasy','Amateur Chef','UltraPro Chef']},
+  ingredients:{type:[String]},
+  cuisine:{type:String,require:true},
+  dishType:{type:String,enum:['breakfast','main_course','soup','snack','drink', 'dessert','other']},
+  image:{type:String,default:"https://images.media-allrecipes.com/images/75131.jpg"},
+  duration:{type:Number},
```

Comment:
In the lab instruction they tell you that `duration` needs to be a `number` and it needs to have the `minimum` time of `0` so you need to add that here:

```
  duration: {
    type: Number,
    min: 0
  },
```